client/components/jetpack/jetpack-product-info/regular-list.tsx

```diff
@@ -1,14 +1,55 @@
-import { FunctionComponent, ReactNode } from 'react';
+import { useRef, useEffect, type FC, type ReactNode } from 'react';
 
 type Props = {
 	items: ReactNode[];
```

Comment:
Do we want some better specificity here? I see we're only working with `Array< TranslateResult >` so perhaps it could be a good compromise. `ReactNode` could be a bit too broad IMHO.

Reply:
I'll have to take a look at other contexts in which this list is component is being used. I've never worked on it before so I want to confirm that we can be more specific than that

Reply:
Yes it seems like that would be correct in all usages, thank you for the improvement suggestion!